server/policies/subscription.ts

```diff
@@ -0,0 +1,109 @@
+import { Subscription, User, Document } from "@server/models";
+import { allow, _cannot as cannot } from "./cancan";
+
+// Is `user` allowed to subscribe to `document`?
+allow(User, "createSubscription", Document, (user, document) => {
+  // Sanity check.
+  if (!document) {
+    return false;
+  }
+
+  // REVIEW: Are viewers allowed to subscribe?
+  if (user.isViewer) {
+    return false;
+  }
+
+  // REVIEW: Show admin be allowed to prescribe
+  // a document to user?
+  if (user.isAdmin) {
+    return true;
+  }
```

Comment:
Not sure if we should allow admins to _**prescribe**_ a subscription.

Reply:
Maybe in the future, lets keep it locked down until the UI exists (it may never exist)